wordle-web/pages/index.vue

```diff
@@ -1,95 +1,77 @@
-              size="75"
-              width="10"
-              :model-value="game.stats.averageGuessesPercent(game.maxAttempts)"
-            >
-              {{ game.stats.averageGuessesPercent(game.maxAttempts).toFixed(0) }} %
-            </v-progress-circular>
-            <br />
-            <i class="text-caption">
-              Average Guesses
-            </i>
-          </v-col>
-        </v-row>
-        <v-btn variant="outlined" @click="game.startNewGame()">
-          <v-icon size="large" class="mr-2"> mdi-restart </v-icon> Restart Game
-        </v-btn>
-      </v-alert>
-      <v-card-title v-else>Wordle</v-card-title>
+    <v-container class="game-container" fluid>
+        <v-card color="primary">
+            <v-card-title>
+                <h1 class="display-1 text-md-center" color="primary">Wordle</h1>
+            </v-card-title>
+            <v-card-text>
+                <p class="display-2 text-md-center" color="primary">Welcome to the Game of Words</p>
+            </v-card-text>
+        </v-card>
+        <v-divider>
+        </v-divider>
+        <v-card-actions>
+            <v-box class="elevation-3" color="primary">
+                <v-btn color="secondary" @click="router.push('../wordOfTheDay')">Wordle - Word of the Day</v-btn>
+            </v-box>
+        </v-card-actions>
+        <v-card-actions>
+            <v-box class="elevation-3" color="primary">
+                <v-btn color="secondary" @click="router.push('../randomWord')">Wordle - Random Word</v-btn>
+            </v-box>
+        </v-card-actions>
+        <v-card-actions>
+            <v-box class="elevation-3" color="primary">
+                <v-btn color="secondary" @click="router.push('../lastTenDays')">Wordle - Last 10 Days</v-btn>
+            </v-box>
+        </v-card-actions>
+    </v-container>
 
-      <GameBoardGuess
-        v-for="(guess, i) of game.guesses"
-        :key="i"
-        :guess="guess"
-      />
-
-      <div class="my-10">
-        <Keyboard />
-      </div>
-
-      <v-btn @click="game.submitGuess()" class="mb-5" color="primary">
-        Guess!
-      </v-btn>
-    </v-card>
-  </v-container>
 </template>
 
 <script setup lang="ts">
-import { Game, GameState } from "../scripts/game";
+import { useDisplay }  from "vuetify";
+const display = useDisplay();
+const imgHeight = ref(630);
+const imgWidth = computed(() => calculateWidth());
+const router = useRouter();
 
-const game = reactive(new Game());
-game.startNewGame();
-provide("GAME", game);
+const calculateWidth = () => {
+  if(display.xs){
+    return '100%';
+  }
+  else if(display.sm){
+    return '90%';
+  }
+  else {
+    return '75%';
+  }
+};
 
-const myGuess = ref("");
+</script>
 
-onMounted(() => {
-  window.addEventListener("keyup", onKeyup);
-});
+<style>
+.image-container{
+  width: 100%;
+}
 
-onUnmounted(() => {
-  window.removeEventListener("keyup", onKeyup);
-});
+.game-container {
+  background-image: url("../public/landingPageImage.jpg");
```

Comment:
I do like this background! I think you could have it extend to all pages (this is how i took the requirements) by removing it from index and putting it in app.vue

Reply:
I have it extended to all pages now, thank you.